kevinb5617/ResuMake
# Create "Select Pages" page

# Description

We need a page which allows users to pick what pages they want on their website (About Me, Resume, etc.)

# Related Issue(s) and Pull Request(s)

closes #14

# Solution

I created a React component for the page "selection", which uses `useState` to "save" the choices to share with the later pages.

# Steps to Test

1. `git checkout feature/create-select-pages`
2. `cd app`
3. `npm install`
4. `npm start`
5. Navigate to `localhost:3000` in your browser.
6. Click the "Get Started" button and you should be brought to the "select pages" page, where you can select/unselect different pages.

# Known Issues

None

# Checklist
